### PR TITLE
New getting started

### PR DESCRIPTION
**Pull request by sarah11918**
REVISED version of new Getting Started landing page. (Will close the previous PR)

Should include all the necessary stuff for Nate's buttons.

### CODE REVIEW

File: astro.config.mjs [hunk 1]
```diff
@@ -3,6 +3,15 @@ export default /** @type {import('astro').AstroUserConfig} */ ({
 	buildOptions: {
 		site: 'https://docs.astro.build/',
 	},
+	markdownOptions: {
```
Comment (FredKSchott) [inline astro.config.mjs:6]: this is what broke the build! Grab the new version (commented out) from the other PR.

File: src/components/Button.astro [hunk 2]
```diff
@@ -0,0 +1,178 @@
+---
+const { class: className = '', style, href } = Astro.props;
+// Wrap in <span> because Houdini is disabled for a[href] for security
+
+const { variant = 'primary' } = Astro.props;
+const { before, after } = Astro.slots;
+---
+
+<!-- Blocking script -->
+<script hoist>
```
Comment (FredKSchott) [inline src/components/Button.astro:10]: I think that this `/pixel.worklet.js` URL 404s, okay to remove the script entirely for now

File: src/pages/en/getting-started-new.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+
+We've made it as easy as possible to get started with Astro either in your browser, or on your machine!
+
+### Online Playgrounds
+
+Visit [astro.new](https://astro.new) for the easiest way to "try before you buy." Choose from a variety of starter templates and start building a full, working version of Astro right in your browser!
+
+Or, **instantly launch our basic starter project** with just one click of a button:
+
+<!-- TODO: Include a link in the first few words of this paragraph to `astro.new`, so that anyone skimming it sees it.
```
Comment (FredKSchott) [inline src/pages/en/getting-started-new.md:20]: left these TODOs for you to take a look at for copy!

File: src/pages/en/getting-started-new.md [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+mkdir my-new-project && cd my-new-project
+npm init astro
+```
+
+⚙️ Our [Installation Guide](/en/installation) has full, step-by-step instructions for installing Astro with your favourite package manager.
+
+⚙️ See instructions for [manual setup](/en/guides/manual-setup) instead.
+
+
+<!-- - TODO: A little blurb about how easy it is to get started with create-astro?
```
Comment (FredKSchott) [inline src/pages/en/getting-started-new.md:44]: some other TODOs!

File: src/pages/en/getting-started-new.md [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+
+### Install Astro Locally
+
+Ready to install?
+
+Get a new project up and running locally in no time with our easy `create-astro` CLI wizard!
+
+```bash
+mkdir my-new-project && cd my-new-project
+npm init astro
```
Comment (FredKSchott) [inline src/pages/en/getting-started-new.md:36]:
```suggestion
# run this command in a new directory to get started!
npm init astro
```

**Review by FredKSchott — COMMENTED** [review]
Left some comments, note that the `TODO` comments in the code are mostly suggestions, feel free to address or ignore how you see fit :)

File: src/pages/en/getting-started-new.md (hunk 3 above)
Comment (sarah11918) [reply src/pages/en/getting-started-new.md:20]: I *think* these todo's were already addressed.... (I might have moved them further away from the things they actually refer to... But every TODO from last night I think I got!)

File: src/pages/en/getting-started-new.md [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+
+📚 Read more about Astro’s [project structure.](/en/core-concepts/project-structure)
+
+📚 Learn more about Astro’s [built-in components.](/en/reference/builtin-components)
+
+📚 Explore Astro’s [API.](/en/reference/api-reference)
+
+*... see more reference material*
+
+## Combine with Astro
```
Comment (FredKSchott) [inline src/pages/en/getting-started-new.md:69]: "Combine" might not be a familiar word here. What about "Explore Astro Integrations"

File: src/pages/en/getting-started-new.md [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+
+🧰 Set up eCommerce.
+
+🧰 Connect a database to your site.
+
+*... see more [third-party integrations](/en/integrations/integrations)*
+
+
+
+## Discuss Astro
```
Comment (FredKSchott) [inline src/pages/en/getting-started-new.md:83]: How about "Join our Community"

File: src/pages/en/getting-started-new.md [hunk 8]
```diff
@@ -0,0 +1,107 @@
...
+
+Join us in the [Astro Discord](https://astro.build/chat) to share with and get help from an active, friendly community!
+
+Say hi in our `#introduce-yourself` channel and let us know who you are!
```
Comment (FredKSchott) [inline src/pages/en/getting-started-new.md:87]: Can these "different things to do" be bullet points, and maybe shorter (<15 words each)?

File: src/pages/en/getting-started-new.md [hunk 9]
```diff
@@ -0,0 +1,107 @@
...
+
+Ask a question in our `#support` channel to automatically receive your own thread and attention from our Support Squad!
+
+Share what you've been working on (no matter how big or how small, finished or in-progress!) in our `#showcase` channel. Ask for feedback, or just soak up the praise! 
+
+
+## Current Version
```
Comment (FredKSchott) [inline src/pages/en/getting-started-new.md:94]: Can this be a "Learn More" section with shortened sub-sections? Something like:

```
## Learn More

- [Our blog](...)
- [Our changelog](...)
- [Migration Guide](...)

**Pull request merged**